src/components/VideoPlayer/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React, {FC} from 'react';
+import ReactPlayerYouTube from 'react-player/youtube';
+import ReactPlayerVimeo from 'react-player/vimeo';
```

Comment:
Only import YouTube and Vimeo players.

@angle943

Reply:
nice